components/aws/sagemaker/tests/integration_tests/utils/sagemaker_utils.py

```diff
@@ -24,3 +24,32 @@ def describe_hpo_job(client, job_name):
 
 def describe_transform_job(client, job_name):
     return client.describe_transform_job(TransformJobName=job_name)
+
+def describe_workteam(client, workteam_name):
+    return client.describe_workteam(WorkteamName=workteam_name)
+
+def list_workteams(client):
+    return client.list_workteams()
+
+def get_cognito_member_definitions(client):
+    # This is one way to get the user_pool and client_id for the Sagemaker Workforce.
+    # An alternative would be to take these values as user input via params or a config file.
```

Comment:
nit: use multiline comments?

Reply:
Done

Reply:
for multiline - you mean docstrings ? Those are supposed to define the how and not the why, hence preferred the regular comments. open to changing.